examples/ephemeral-environments/README.md

```diff
@@ -0,0 +1,124 @@
+# Ephemeral Environments in rok8s-scripts
+
+This is an example of how rok8s-scripts could be used to deploy ephemeral environments to your Kubernetes
+cluster using helm. These ephemeral environments will be created from any feature branches that are pushed
+to the source repository based on a regex match.
+
```

Comment:
Would you add a summary here? Something like :

"In the example described below, branches that match the regex pattern `feature-.*` trigger CircleCI to use that branch to create a new namespace and deploy to it as configured by the `./deploy/ephemeral.config`  file. Where ephemeral environments require other processes you can deploy the other process with a matching branch name or use a something like a helm chart to deploy multiple processes when they are required by the Ephemeral Environment"

Otherwise, I think a few key ideas are left out.

Reply:
I like it. Done.